app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">PetOwner</string>
```

Comment:
Should we add a space between _Pet_ and _Owner_ to improve readability?

Reply:
Changed